moped-editor/src/views/projects/projectView/DateFieldEditComponent.js

```diff
@@ -9,19 +10,22 @@ import { handleKeyEvent } from "../../../utils/materialTableHelpers";
  * @constructor
  */
 
-const DateFieldEditComponent = (props) => (
-  <TextField
-    name={props.name}
-    label={props.label}
-    type="date"
-    variant="standard"
-    value={props.value || ""}
-    onChange={(e) => props.onChange(e.target.value)}
-    onKeyDown={(e) => handleKeyEvent(e)}
-    InputLabelProps={{
-      shrink: true,
-    }}
-  />
-);
+const DateFieldEditComponent = (props) => {
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+      <DatePicker
+        clearable={true}
+        emptyLabel="mm/dd/yyyy"
+        format="MM/dd/yyyy"
+        value={props.value ? parseISO(props.value) : null}
+        onChange={(e) => {
+          const newDate = e ? format(e, "yyyy-MM-dd") : null;
+          props.onChange(newDate);
+        }}
```

Comment:
Not a super strong opinion but this onChange handler function could be defined in body of this component

Reply:
I just added this, thanks for the comment I totally agree!